react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -180,6 +152,23 @@ export default {
     },
 
     /**
+     * The descriptor of the dial out toolbar button.
+     */
+    dialOut: {
+        classNames: [ 'button', 'icon-telephone' ],
+        enabled: true,
+
+        // Will be displayed once the SIP calls functionality is detected.
+        hidden: true,
+        id: 'toolbar_button_dial_out',
+        onClick() {
+            JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
+            APP.store.dispatch(openDialOutDialog());
```

Comment:
While I think this is true I'm still unsure of a way to do so given the current creation of buttons done programmatically instead of declaratively. Ilya was going to get dispatch into the onClick function through a closure: https://github.com/jitsi/jitsi-meet/pull/1498/files. The way I got around it for audio only was by creating a button component.